.travis.yml

```diff
@@ -23,6 +22,8 @@ jobs:
         - python3 codalab_service.py build services --version ${TRAVIS_BRANCH} --pull $([ -z "${CODALAB_DOCKER_USERNAME}" ] || echo "--push")
       script:
         - python3 codalab_service.py start --services default monitor test --version ${TRAVIS_BRANCH}
+        - python3 codalab_service.py stop
+        - python3 codalab_service.py start --services default monitor test --version ${TRAVIS_BRANCH} --shared-file-system
```

Comment:
We're running the tests again?  This is going to make our builds really slow...

Reply:
Just created #1536 so I can change this to only run worker-related tests

Reply:
Hmm yes, after those changes, this can just restart the worker.